elide-core/src/main/java/com/yahoo/elide/graphql/PersistentResourceFetcher.java

```diff
@@ -0,0 +1,244 @@
+            throw new WebApplicationException("Attempted to delete non-existent id.", HttpStatus.SC_BAD_REQUEST);
+        }
+
+        deleteObject.deleteResource();
+        deleted.add(deleteObject);
+
+        return deleted;
+    }
+
+    private Object createObject(Environment request) {
+        EntityDictionary dictionary = request.requestScope.getDictionary();
+
+        GraphQLObjectType objectType;
+        String uuid = UUID.randomUUID().toString();
+        if (request.outputType instanceof GraphQLObjectType) {
+            // No parent
+            // TODO: These UUID's should not be random. They should be whatever id's are specified by the user so they
+            // can be referenced throughout the document
+            objectType = (GraphQLObjectType) request.outputType;
+            return PersistentResource.createObject(null, dictionary.getEntityClass(objectType.getName()),
+                    request.requestScope, uuid);
+
+        } else if (request.outputType instanceof GraphQLList) {
+            // Has parent
+            objectType = (GraphQLObjectType) ((GraphQLList) request.outputType).getWrappedType();
+            List<PersistentResource> container = new ArrayList<>();
+            for (Map<String, Object> input : request.data) {
+                Class<?> entityClass = dictionary.getEntityClass(objectType.getName());
+                // TODO: See above comment about UUID's.
+                PersistentResource toCreate = PersistentResource.createObject(null, entityClass, request.requestScope,
+                        uuid);
+                input.entrySet().stream()
+                        .filter(entry -> dictionary.isAttribute(entityClass, entry.getKey()))
+                        .forEach(entry -> toCreate.updateAttribute(entry.getKey(), entry.getValue()));
+                container.add(toCreate);
+            }
+            return container;
+        }
+        throw new IllegalStateException("Not sure what to create " + request.outputType.getName());
+    }
+
+    private Object fetchObject(Environment request) {
+        if (!request.data.isEmpty()) {
+            throw new WebApplicationException("FETCH must not include data.", HttpStatus.SC_BAD_REQUEST);
+        }
+
+        Optional<FilterExpression> filters = Optional.empty();
+        Optional<Sorting> sorting = Optional.empty();
+        Optional<Pagination> pagination = Optional.empty();
+
+        RequestScope requestScope = request.requestScope;
+        if (request.outputType instanceof GraphQLList) {
+            if (request.id.isPresent() && request.filters.isPresent()) {
+                throw new WebApplicationException("You may not filter when loading by id");
+            }
+
+            GraphQLObjectType graphQLType = (GraphQLObjectType) ((GraphQLList) request.outputType).getWrappedType();
+            String entityType = graphQLType.getName();
+            Class recordType = (Class) requestScope.getDictionary().getEntityClass(entityType);
```

Comment:
This (incorrectly) makes the assumption that a List implies the element type is an Elide entity.  We could have an attribute that is a list of integers.